File: src/targets/csharp/httpclient/fixtures/short.cs
using System.Net.Http.Headers;
var client = new HttpClient();
var request = new HttpRequestMessage
{
    Method = HttpMethod.Get,
<<<<<<< HEAD
    RequestUri = new Uri("https://httpbin.org/anything"),
=======
    RequestUri = new Uri("http://httpbin.org/anything"),
>>>>>>> upstream/master
};
using (var response = await client.SendAsync(request))
{
    response.EnsureSuccessStatusCode();
    var body = await response.Content.ReadAsStringAsync();
    Console.WriteLine(body);
}

File: src/targets/node/request/fixtures/short.js
const request = require('request');

<<<<<<< HEAD
const options = {method: 'GET', url: 'https://httpbin.org/anything'};
=======
const options = {method: 'GET', url: 'http://httpbin.org/anything'};
>>>>>>> upstream/master

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});